auth-web/src/components/auth/AddNameRequestForm.vue

```diff
@@ -0,0 +1,247 @@
+            </li>
+          </ul>
+          <div>
+            <p class="mb-0"><strong>Hours of Operation:</strong><br>Monday to Friday, 8:30am - 4:30pm <span title="Pacific Standard Time">PST</span></p>
+          </div>
+        </v-card-text>
+        <v-card-actions>
+          <v-spacer></v-spacer>
+          <v-btn large color="primary" @click="helpDialog = false">OK</v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+  </div>
+</template>
+
+<script lang="ts">
+import { Business, FolioNumberload, LoginPayload, UpdateFilingBody } from '@/models/business'
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { mapActions, mapMutations, mapState } from 'vuex'
+import BusinessModule from '@/store/modules/business'
+import CommonUtils from '@/util/common-util'
+import ConfigHelper from '@/util/config-helper'
+import { CreateNRAffiliationRequestBody } from '@/models/affiliation'
+import { Organization } from '@/models/Organization'
+import { getModule } from 'vuex-module-decorators'
+import { mask } from 'vue-the-mask'
+
+@Component({
+  directives: {
+    mask
+  },
+  computed: {
+    ...mapState('org', ['currentOrganization'])
+  },
+  methods: {
+    ...mapActions('business', [
+      'addNameRequest',
+      'updateFiling',
+      'updateFolioNumber'
+    ])
+  }
+})
+export default class AddNameRequestForm extends Vue {
+  private readonly currentOrganization!: Organization
+  private readonly addNameRequest!: (payload: CreateNRAffiliationRequestBody) => any
+  private readonly updateFiling!: (filingBody: UpdateFilingBody) => any
+  private readonly updateFolioNumber!: (folioNumberload: FolioNumberload) => void
+  private validationError = ''
+  private entityNumRules = [
+    v => !!v || 'Name Request Number is required',
+    v => CommonUtils.validateNameRequestNumber(v) || 'Name Request Number is invalid'
+  ]
+  private entityPhoneNumberRules = [
+    v => this.isInputEntered(v, 'phone') || 'Phone number is required',
+    v => !(v.length > 12) || 'Phone number is invalid'
+  ]
+  private entityEmailRules = [
+    v => this.isInputEntered(v, 'email') || 'Email is required',
+    v => this.isValidateEmail(v) || 'Email is Invalid'
+  ]
+  private VUE_APP_COPS_REDIRECT_URL = ConfigHelper.getValue('VUE_APP_COPS_REDIRECT_URL')
+  private nameRequestNumber: string = ''
+  private applicantPhoneNumber: string = ''
+  private applicantEmail: string = ''
+  private passcode: string = ''
+  private folioNumber: string = ''
+  private helpDialog = false
+  private isLoading = false
+
+  $refs: {
+    addNRForm: HTMLFormElement
+  }
+
+  private isFormValid (): boolean {
+    return !!this.nameRequestNumber &&
+      (!!this.applicantPhoneNumber || !!this.applicantEmail)
+  }
+
+  private isInputEntered (value: any, ipType: string) {
```

Comment:
ip means input rite?..fir a moment , i thought its something else...  may be just call it as inputType.

Reply:
will change